vaadin-flow-components-shared-parent/vaadin-flow-components-base/pom.xml

```diff
@@ -27,7 +27,12 @@
     <dependency>
       <groupId>jakarta.servlet</groupId>
       <artifactId>jakarta.servlet-api</artifactId>
-  </dependency>
+    </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Doesn't look like this is needd anymore.

Reply:
It is still used in 3 of the unit tests (the ones for initializing disabled on click).

Reply:
Right, sorry, I was looking at the wrong module.